.github/workflows/phase_1_python.yml

```diff
@@ -0,0 +1,232 @@
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Trivy
+        run: |
+          curl -L -o /tmp/trivy.tgz \
+            "https://github.com/aquasecurity/trivy/releases/download/v${TRIVY_VERSION}/trivy_${TRIVY_VERSION}_Linux-64bit.tar.gz"
+          tar xvf /tmp/trivy.tgz -C /tmp
+          chmod +x /tmp/trivy
+
+      - name: "CycloneDX: Generate SBOM"
+        working-directory: "Phase 1/Python"
+        run: |
+          /tmp/trivy fs \
+            --format cyclonedx \
+            --output /tmp/application-sbom.cdx.json \
+            requirements.txt
+
+      - name: "SPDX: Generate SBOM"
+        working-directory: "Phase 1/Python"
+        run: |
+          /tmp/trivy fs \
+            --format spdx-json \
+            --output /tmp/application-sbom.spdx.json \
+            requirements.txt
+
+      - name: Upload CycloneDX SBOM
+        uses: actions/upload-artifact@v4
+        with:
+          name: application-sbom-cyclonedx
+          path: "/tmp/application-sbom.cdx.json"
+
+      - name: Upload SPDX SBOM
+        uses: actions/upload-artifact@v4
+        with:
+          name: application-sbom-spdx
+          path: "/tmp/application-sbom.spdx.json"
+
+  Enrich:
+    runs-on: ubuntu-latest
+    needs: [Container, Application]
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Download all workflow run artifacts
+        uses: actions/download-artifact@v4
+
+      - name: Enrich Container CycloneDX
+        run: |
+          # We're using `envsubst` here to populate the metadata
+          # template from environment variables
+          export SBOM_NAME='phase1-python-container'
+          cat "Phase 1/Python/sbom/metadata.cdx.json.tmpl" | jq | \
+            envsubst > metadata.cdx.json
+          unset SBOM_NAME
+
+          # Merge together SBOM and metadata
+          jq --slurp '.[0] * .[1]' \
+            container-sbom-cyclonedx/container-sbom.cdx.json \
+            metadata.cdx.json \
+            > /tmp/enriched_container-sbom.cdx.json
+
+      - name: Enrich Application CycloneDX
+        run: |
+          # We're using `envsubst` here to populate the metadata
+          # template from environment variables
+          export SBOM_NAME='phase1-python-application'
+          cat "Phase 1/Python/sbom/metadata.cdx.json.tmpl" | jq | \
+            envsubst > metadata.cdx.json
+          unset SBOM_NAME
+
+          # Merge together SBOM and metadata
+          jq --slurp '.[0] * .[1]' \
+            application-sbom-cyclonedx/application-sbom.cdx.json \
+            metadata.cdx.json \
+            > /tmp/enriched_application-sbom.cdx.json
+
+      - name: Enrich Container SPDX
+        run: |
+          # We're using `envsubst` here to populate the metadata
+          # template from environment variables
+          export SBOM_NAME='phase1-python-container'
+          cat "Phase 1/Python/sbom/metadata.spdx.json.tmpl" | jq | \
+            envsubst > metadata.spdx.json
```

Comment:
I think jq is having trouble with the template file:

``` sh
cat "Phase 1/Python/sbom/metadata.spdx.json.tmpl" | jq
parse error: Expected another key-value pair at line 3, column 1
```

which results in this Actions error:

<img width="628" alt="Screenshot 2024-08-27 at 5 30 14 PM" src="https://github.com/user-attachments/assets/c1b6f308-b59a-451a-8902-c0f6f96d9fa7">